itou/companies/management/commands/_import_siae/__init__.py

```diff
@@ -15,5 +16,10 @@ def gunzip(archivepath, outdir):
         raise shutil.ReadError from e
 
 
+def unpack_riae_zip_aes_encrypted(path, directory, **kwargs):
+    with pyzipper.AESZipFile(path) as zf:
+        zf.extractall(directory, pwd=os.getenv("ASP_RIAE_UNZIP_PASSWORD", "").encode())
```

Comment:
Autant que le fait que la variable d’environnement est obligatoire soit clair, non ?
```suggestion
        zf.extractall(directory, pwd=os.environ["ASP_RIAE_UNZIP_PASSWORD"].encode())
```

Reply:
Yep, c'est un reste de quand je voulais faire un truc "propre" et réutilisable mais vu qu'on peux pas passer d'argument à `shutil.unpack_archive()` bah c'est du dur :sweat:.